### PR TITLE
Removal of self-signed certificates, usage of AWS certificates

### PR DESCRIPTION
**Pull request by angeloskyratzakos**
#### Summary
Update terraform to accommodate AWS ACM certificates and discard custom-made certificates/variables

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-22902

### CODE REVIEW

**Comment by cpanato** [comment]
@angeloskyratzakos can you make the title of the PR more accurate? :)

**Comment by angeloskyratzakos** [comment]
Sorry @cpanato , I have updated it

File: terraform/aws/environments/test/4-command-and-control/3-cluster-post-installation/main.tf [hunk 1]
```diff
@@ -5,6 +5,9 @@ terraform {
     key    = "mattermost-central-command-control-post-installation"
     region = "us-east-1"
   }
+  required_providers {
+    helm = "~> 0.10"
+  }
```
Comment (stylianosrigas) [inline terraform/aws/environments/test/4-command-and-control/3-cluster-post-installation/main.tf:10]: @angeloskyratzakos can we add the pin down for staging and prod as well?

Comment (angeloskyratzakos) [reply terraform/aws/environments/test/4-command-and-control/3-cluster-post-installation/main.tf:10]: Done I have pinned down prod and staging as well.

**Pull request merged**